cg/test/management/users.api.spec.js

```diff
@@ -154,3 +154,70 @@ describe('User password update', () => {
     expect((await res2.json()).jwt).toBeTruthy();
   });
 });
+
+describe('Listing management users', () => {
+  it('Should display an empty list when no manager is registered', async () => {
+    //Given
+    await db('users').del(); // It's fine to delete the contents of the managers table because no other table references it
+
+    //When
+    const res = await fetch('/api/management/users/list', {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${managementToken}`
+      }
+    });
+
+    //Then
+    expect(res.ok).toBeTruthy();
+    expect(res.status).toBe(200);
+    expect(await res.json()).toEqual(expect.arrayContaining([]));
+  });
+
+  it('Should display correctly the registered managers', async () => {
+    //Given
+    await db('users').del(); // It's fine to delete the contents of the managers table because no other table references it
+    const managementToken1 = await managementJWT.sign({ id: '1' });
+    const managementToken2 = await managementJWT.sign({ id: '2' });
```

Comment:
You don't need a second token, `managementToken1` can be reused for both requests. It's basically a case when single manager with ID 1 creates two users. What you implemented is manager 1 and manager 2 are creating 1 user each.

Reply:
Ok